invisible_cities/core/configure_test.py

```diff
@@ -15,9 +13,14 @@
 from . exceptions import NoInputFiles
 from . exceptions import NoOutputFile
 
+from .. types .symbols     import EventRange
 from .. cities.components  import city
 from .. cities.penthesilea import penthesilea
 
+all  = EventRange.all
```

Comment:
I think I just did it to keep the definitions as they were before this PR so the relevant change was more obvious, but I will take a look and if there is no reason to put it like this I will replace it with `EventRange.all` instead of `all`